### PR TITLE
Updated readmes with Arm64 info.

### PR DESCRIPTION
**Pull request by bryanjhogan**
Added x86_64 explicitly to config

*Issue #, if available:*

*Description of changes:*
Added Arm64 instructions to readme for two templates.
Added x86_64 as explicitly to config for two templates. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction/template/src/BlueprintBaseName.1/Readme.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ Deploy function to AWS Lambda
 
 ## Arm64
 
-If you want to run your Lambda on an Arm64 processor, all you need is to do is add `"function-architecture": "arm64"` to the `aws-lambda-tools-defaults.json` file. Then deploy as described above.
+Graviton2 processors offer the best performance and price for running Lambda functions.
```
Comment (normj) [inline Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction/template/src/BlueprintBaseName.1/Readme.md:47]: I feel like this statement is too absolute and sounds too much like marketing for a instruction README file. Graviton2 is great and I know it is often the best performance/price for running Lambda but I don't think we can say that is always the best.

What do you think of this?

`Arm64 support is provided by the AWS Graviton 2 processor. For many Lambda work loads Graviton 2 provides the best price performance.`

File: Blueprints/BlueprintDefinitions/vs2022/EmptyServerless/template/src/BlueprintBaseName.1/Readme.md [hunk 2]
```diff
@@ -40,3 +40,8 @@ Deploy application
     cd "BlueprintBaseName.1/src/BlueprintBaseName.1"
     dotnet lambda deploy-serverless
 ```
+## Arm64
+
+Graviton2 processors offer the best performance and price for running Lambda functions.
```
Comment (normj) [inline Blueprints/BlueprintDefinitions/vs2022/EmptyServerless/template/src/BlueprintBaseName.1/Readme.md:45]: Same comment as above.

**Review by normj — CHANGES_REQUESTED** [review]

File: Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction/template/src/BlueprintBaseName.1/Readme.md (hunk 1 above)
Comment (bryanjhogan) [reply Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction/template/src/BlueprintBaseName.1/Readme.md]: Updated as requested.

**Review by normj — APPROVED** [review]
Change looks good. Do you want to make the same changes in all of the VS 2019 blueprints or do that as a separate PR?

**Review by philasmar — APPROVED** [review]

**Pull request merged**